runtime/compiler/env/j9method.cpp

```diff
@@ -6103,6 +6164,12 @@ TR_ResolvedJ9Method::varHandleMethodTypeTableEntryAddress(int32_t cpIndex)
    return ramClass->varHandleMethodTypes + methodTypeIndex;
    }
 
+bool
```

Comment:
This new method is used in `9::SymbolReferenceTable::findOrCreateVarHandleMethodTypeTableEntrySymbol(TR::ResolvedMethodSymbol * owningMethodSymbol, int32_t cpIndex)`
Could you please change `J9SymbolReferenceTable.cpp` to use it? Otherwise we might forget.